packages/MJCore/src/generic/baseEntity.ts

```diff
@@ -1117,5 +1122,20 @@ export abstract class BaseEntity {
             else
                 return [];
         }
-    } 
+    }
+
+    /**
+     * Strongy-typed wrapper for the {@link SetMany} method.
+     */
+    public From(baseType: T, replaceOldValues?: boolean): void {
+        this.init();
+        this.SetMany(baseType, false, replaceOldValues);
+    }
+
+    /**
+     * Strongly-typed wrapper for the {@link GetAll} method
+     */
+    public To(): T {
+        return this.GetAll() as T;
+    }
```

Comment:
I think this should accept a schema as an argument so the fields are legit validated– using `as T` basically opts out of type checking and there's no runtime validation

Reply:
Maybe we have an optional variable for this instead of making it required and if the schema is provided only then do we validate?